extensions/wikia/Search/classes/ProfiledClassFactory.php

```diff
@@ -81,34 +89,47 @@ class {$terminal}Profiled extends {$terminal}
 						$default = $parameter->getDefaultValue();
 					}
 				}
-				$sig = trim( str_replace( '<optional>', '', preg_replace( '/^.*\[ (<required>)?([^\]]+) \]/', '$2', ReflectionParameter::export( array( $method->class, $method->name ), $parameter->getName(), true ) ) ) );
-				
+				$sig = trim(
+					str_replace(
+						'<optional>',
+						'',
+						preg_replace(
+							'/^.*\[ (<required>)?([^\]]+) \]/',
+							'$2',
+							ReflectionParameter::export(
+								[ $method->class, $method->name ],
+								$parameter->getName(),
+								true
+							)
+						)
+					)
+				);
+
 				if ( $sig[0] != '$' && $sig[0] != '\\' && substr_count( $sig, 'array', 0, 5 ) == 0 ) {
 					$sig = '\\' . $sig;
 				}
 				if ( substr_count( $sig, '= Array' ) > 0 ) {
-					$sig = str_replace( '= Array', ' = '.$default, $sig );
+					$sig = str_replace( '= Array', ' = ' . $default, $sig );
 				}
-				if ( substr_count( $sig, ' or NULL') > 0 ) {
+				if ( substr_count( $sig, ' or NULL' ) > 0 ) {
 					$sig = str_replace( ' or NULL', '', $sig );
 				}
 				$paramSigs[] = $sig;
 			}
 			$paramSignature = implode( ', ', $paramSigs );
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "$paramSignature" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1481)